ios/Classes/PTNavigationController.m

```diff
@@ -13,11 +13,15 @@ - (void)setNavigationBarHidden:(BOOL)hidden animated:(BOOL)animated
 
 - (void)setToolbarHidden:(BOOL)hidden animated:(BOOL)animated
 {
-    BOOL allowed = [self.flutterDocumentController shouldSetToolbarHidden:hidden animated:animated];
+    BOOL allowed = [[self getCurrentFlutterDocumentController] shouldSetToolbarHidden:hidden animated:animated];
```

Comment:
This could have been a property, where the property's implementation ran the code as in this "getter". It's more idiomatic on iOS programming to have it as a property.